sentences/en/_common.yaml

```diff
@@ -361,28 +361,36 @@ lists:
     wildcard: true
 
 expansion_rules:
-  name: "[the] {name}"
-  area: "[the] {area}"
-  floor: "[the] {floor}[ floor]"
-  what_is: "(what's|whats|what is)"
+  name: "[the ]{name}"
+  area: "[the ]{area}"
+  floor: "[the ]{floor}[ floor]"
+  area_floor: "(<area>|<floor>)"
+  in_area_floor: "[<in> ]<area_floor>"
+  what_is: "(what's|whats|what is|tell me)[ the]"
   where_is: "(where's|wheres|where is)"
+  which: "(which|what)[ of the]"
+  is: "(is|are)[ (there|the)]"
+  are: "<is>"
+  any: "[<are> ](any|some)[ of the]"
+  how_many: "how many[ of the]"
   brightness: "{brightness}[[ ]%| percent]"
-  light: "[the] (light|lights|lighting)"
+  light: "[the ](light|lights|lighting|lamp|lamps)"
   turn: "(turn|switch|change)"
   temp: "(temp|temperature)"
   temperature: "{temperature}[([ ]°[ ][{temperature_unit}])|( degrees[ {temperature_unit}])]"
-  open: "(open|raise)"
-  close: "(close|shut|lower)"
+  open: "(open|raise|lift)[ up]"
+  close: "(close|shut|lower)[ (up|down)]"
```

Comment:
Can you provide a few examples where `up` makes sense?

Reply:
"Ok Nabu, open up the kitchen shades"

Maybe this is a dialect thing, but throwing unnecessary prepositions is definitely something commonly done in the Chicago-land area where I grew up.

Reply:
I meant for `close`

Reply:
Ah. That kind of depends on context. But ["Close up the house"](https://www.wordnik.com/words/close%20up) is something I've said/heard. I can't imagine someone using it with `lower`, but `close up` and `shut up` are certainly used.

Reply:
I won't argue with a native English speaker on this one, I'll just tale your word for it 😅